src/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Button() {
+
+    return(
+        <div className="search-row">
+            <input
```

Comment:
This isn't quite right I think - the component is named "Button" but it also includes the input box and the wrapping div.